content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -259,45 +315,19 @@ beforeEach(() => {
 #### Execution
 
 Cypress executes the support file before the spec file. For example when you
-click on a test file named `spec-a.js` via
+click on a test file named `spec-a.cy.js` via
 [cypress open](/guides/guides/command-line#cypress-open), then the Cypress App
 executes the files in the following order:
```

Comment:
I think we should also mention `cypress run` here, something like:

> Cypress executes the support file before the spec file. For example, when cypress executes a spec file via [`cypress open`](/guides/guides/command-line#cypress-open) or [`cypress run`](/guides/guides/command-line#cypress-run), it executes the files in the following order:

Reply:
I think that reads better. It's updated in this commit https://github.com/cypress-io/cypress-documentation/pull/4328/commits/f3b0e6a3d368318377d078945ee22fb661cbff39